### ISSUE TITLE
Client vs Server patch

### ISSUE THREAD

**Issue by SpawnChunks** [gnembon/carpetmod#12]
Hello there,
Is the client patch for single player and the server patch for the Minecraft server jar file?
Or is the client patch for client side effects like moving pistonheads, and stuff so it appears slower when the tickrate is turned down?
Thanks.

**Comment by gnembon** [comment]
Correct: server patch is to apply it to the server jar and that's primary use of the carpet mod. However with the new build tool from Earthcomputer - we get automatically a client patch to play singleplayer (even with other fake players its still singleplayer). Some animations run smoother but in general we didn't do anything with the animations wrt different tickrate.
